README.md

```diff
@@ -121,13 +123,13 @@ Once you're done with the above setup (no matter which method you used), run `np
     Checking theme for possible violations:
 
     ```bash
-    wpcs wp-content/themes/inf_theme
+    wpcs wp-content/themes/f
     ```
 
     Autofix theme for minor violations:
 
     ```bash
-    wpcbf wp-content/themes/inf_theme
+    wpcbf wp-content/themes/f
     ```
 
 ## Goint to Production
```

Comment:
The two scripts should be `build.sh` and `deploy.sh` right?

Reply:
I couldn't mark the lines below, the readme has 2x `bash bin/build.sh` mentioned for build and deploy. And it should be `bash bin/build.sh` and `bash bin/deploy.sh` 🙂

Reply:
They're currently the same so I'm guessing that's why it was written like so. Will change tho. 👍